editor/test/editor/process_test.clj

```diff
@@ -0,0 +1,40 @@
+(ns editor.process-test
+  (:require [clojure.test :refer :all]
+            [editor.process :as process])
+  (:import [java.io ByteArrayOutputStream]))
+
+(deftest start-test
+  (let [p (process/start "ls")
```

Comment:
Does Windows have the `ls` command now? They used to have `dir` instead?

Reply:
Hmm no, the tests will fail on windows. I'll make sure these tests are skipped on windows.